source/reflect.h2

```diff
@@ -344,12 +351,33 @@ function_declaration: @copyable type =
     has_non_void_return_type     : (this) -> bool = n*.has_non_void_return_type();
 
     unnamed_return_type          : (this) -> std::string = n*.unnamed_return_type_to_string();
+    get_parameters: (this)
+        -> std::vector<object_declaration>
+    = {
+        ret: std::vector<object_declaration> = ();
+        for n*.get_function_parameters() do (d) {
+            _ = ret.emplace_back( d*.declaration*&, this );
+        }
+        return ret;
+    }
 
     is_binary_comparison_function: (this) -> bool = n*.is_binary_comparison_function();
 
     default_to_virtual           : (inout this) = _ = n*.make_function_virtual();
 
     make_virtual                 : (inout this) -> bool = n*.make_function_virtual();
+
+    add_initializer: (move this, source: std::string_view)
```

Comment:
Yes, it was.
It's can be considered a manifestation of [Value category is not lifetime](https://quuxplusone.github.io/blog/2019/03/11/value-category-is-not-lifetime/).
Since my implementation did remove the member,
this was necessary as an indication of safety.

Reply:
Oh right, and I even briefly thought about that move-from-this-member original characteristic, then forgot it when writing the comment. OK, anyway it's good now, thanks. 👍